pkg/controller/endpointslicemirroring/reconciler_helpers_test.go

```diff
@@ -138,6 +139,13 @@ func expectEqualSlices(t *testing.T, actual, expected []*discovery.EndpointSlice
 		t.Fatalf("Expected %d EndpointSlices, got %d: %v", len(expected), len(actual), actual)
 	}
 
+	sort.Slice(actual, func(i, j int) bool {
+		return actual[i].Name < actual[j].Name
+	})
```

Comment:
suggest moving this to line 127 and sorting `t.startingSlices.{toCreate,toUpdate,toDelete}`, and avoiding mutating expectedSlices (we can put those in sorted order in the test setup)